duo_admin/help.md

```diff
@@ -318,18 +318,23 @@ Example input:
 This action is used to get auth logs, limited to past 180 days.
 [Currentmillis.com](https://currentmillis.com/) is useful for finding a usable UNIX timestamp.
 
+Available inputs for parameters:
+
+* `factors` - a comma-separated list of factors(duo_push, phone_call, u2f_token, hardware_token, bypass_code, sms_passcode, duo_mobile_passcode, yubikey_code, passcode, digipass_go_7_token, not_available, sms_refresh, remembered_device, trusted_network), if left empty, the action returns the authentication logs for all factors used for an authentication attempt
```

Comment:
Can you indents within the unordered list to make the available inputs more readable? https://www.markdownguide.org/basic-syntax#unordered-lists

Reply:
@mrinehart-r7 Done :)